### PR TITLE
[fix](multi-table) fix unknown source slot descriptor when load multi table

### PR DESCRIPTION
**Pull request by sollhui**
### Bug
image

when use one single multi table load to load data to multi tables, it can not load successfully for unknown source slot descriptor.

### Reason
The reason why 'unknown source slot descriptor' is multi table shared one descriptor when load.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java
Comment (freemandealer) [inline fe/fe-core/src/main/java/org/apache/doris/service/FrontendServiceImpl.java:1]: innecessary change

File: be/src/pipeline/pipeline_fragment_context.cpp [hunk 1]
```diff
@@ -249,13 +249,18 @@ Status PipelineFragmentContext::prepare(const doris::TPipelineFragmentParams& re
         fragment_context->set_is_report_success(request.query_options.is_report_success);
     }
 
-    auto* desc_tbl = _query_ctx->desc_tbl;
-    _runtime_state->set_desc_tbl(desc_tbl);
+    if (request.is_simplified_param) {
+        _desc_tbl = _query_ctx->desc_tbl;
+    } else {
+        RETURN_IF_ERROR(
```
Comment (freemandealer) [inline be/src/pipeline/pipeline_fragment_context.cpp:255]: DCHECK request.__isset.desc_tbl

**Review by freemandealer — CHANGES_REQUESTED** [review]

**Comment by HHoflittlefish777** [comment]
run buildall

**Review by freemandealer — APPROVED** [review]
LGTM

**Comment by HHoflittlefish777** [comment]
run clickbench

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**